tests/unit/client.py

```diff
@@ -0,0 +1,442 @@
+    """
+
+    # Adding nodes, node-01 - node-06
+    url_node = 'http://127.0.0.1:8888/node/'
+    api_nodename = 'http://schema.massopencloud.org/haas/v0/obm/'
+
+    obminfo1 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.01",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo2 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.02",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo3 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.03",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo4 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.04",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo5 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.05",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo6 = {
+            "type": api_nodename+'ipmi', "host": "10.10.0.06",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo7 = {
+            "type": api_nodename+'mock', "host": "10.10.0.07",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    obminfo8 = {
+            "type": api_nodename+'mock', "host": "10.10.0.08",
+            "user": "ipmi_u", "password": "pass1234"
+            }
+
+    requests.put(url_node+'node-01', data=json.dumps({"obm": obminfo1}))
+    requests.put(url_node+'node-02', data=json.dumps({"obm": obminfo2}))
+    requests.put(url_node+'node-03', data=json.dumps({"obm": obminfo3}))
+    requests.put(url_node+'node-04', data=json.dumps({"obm": obminfo4}))
+    requests.put(url_node+'node-05', data=json.dumps({"obm": obminfo5}))
+    requests.put(url_node+'node-06', data=json.dumps({"obm": obminfo6}))
+    requests.put(url_node+'node-07', data=json.dumps({"obm": obminfo7}))
+    requests.put(url_node+'node-08', data=json.dumps({"obm": obminfo8}))
+
+    # Adding Projects proj-01 - proj-03
+    for i in ["proj-01", "proj-02", "proj-03"]:
+        requests.put('http://127.0.0.1:8888/project/'+i)
+
+    # Adding switches one for each driver
+    url = 'http://127.0.0.1:8888/switch/'
+    api_name = 'http://schema.massopencloud.org/haas/v0/switches/'
+
+    dell_param = {
+            'type': api_name+'powerconnect55xx', 'hostname': 'dell-01',
+            'username': 'root', 'password': 'root1234'
+            }
+    nexus_param = {
+            'type': api_name+'nexus', 'hostname': 'nexus-01',
+            'username': 'root', 'password': 'root1234', 'dummy_vlan': '333'
+            }
+    mock_param = {
+            'type': api_name+'mock', 'hostname': 'mockSwitch-01',
+            'username': 'root', 'password': 'root1234'
+            }
+    brocade_param = {
+            'type': api_name+'brocade', 'hostname': 'brocade-01',
+            'username': 'root', 'password': 'root1234',
+            'interface_type': 'TenGigabitEthernet'
+            }
```

Comment:
You could make this a list and then turn the function calls below into a for loop.